.circleci/config.yml

```diff
@@ -282,15 +289,23 @@ jobs:
 
 
   test_integration:
-    executor: node
-    environment:
-      SAUCE_TUNNEL_ID: lwc_<< pipeline.id	>>_test_integration
+    executor: node-browsers
     steps:
+      - browser-tools/install-browser-tools
       - load_workspace
-      - start_sauce_connect
```

Comment:
I missed the details during the PR review chat but it would be good to document here why no sauce_labs, considering the other two `test_integration_*` jobs do use it.